### PR TITLE
Create NexClipper-Agent examples

### PR DESCRIPTION
**Pull request by ssipflow**
Create example for nexclipper-agent

### CODE REVIEW

File: nexclipper-agent/1.10/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# How to use NexClipper-Agent with DC/OS
+
+[NexClipper][nexclipper] is the container monitoring and performance management service specialized in Docker, Apache Mesos, Marathon, DC/OS, Mesosphere, Kubernetes. In this beta version, it support fundamental function to monitor and analyze container cluster(Mesos, DC/OS, Mesosphere, Kubernetes), Container, Agent, event management, resource tracing, performance monitor and basic root cause functionality for event management with SaaS version. It provide more rich functionality for root cause (event management, resource tracing, performance monitor and transaction tasing, etc) soon.
```
Comment (mattj-io) [inline nexclipper-agent/1.10/README.md]: is a container monitoring and performance management service for Docker, Apache Mesos, Marathon, DC/OS, Mesosphere, Kubernetes. In this beta version, it supports monitoring and analysis of the container cluster (Mesos, DC/OS, Mesosphere, Kubernetes), Container, Agent, event management, resource tracing, performance monitor and basic root cause functionality for event management with SaaS version. Future versions will provide rich functionality for root cause (event management, resource tracing, performance monitor and transaction tasing, etc) soon.

File: nexclipper-agent/1.10/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Installation on DC/OS agent nodes
+
+Go to the DC/OS universe/catalog web UI and search for "NexClipper-Agent". Click the tile and select "Configure" to enter the required parameters for connecting with NexCloud.
+
+Click the tile and select "Configure" to enter the required parameters for connecting with NexClipper-Agent.
+
+1. Get your NexClipper-Agent key Download URL from your [NexClipper][nexclipper] environment
+
+3. Set the agnet-key you downloaded
```
Comment (mattj-io) [inline nexclipper-agent/1.10/README.md]: agent key

**Review by mattj-io — CHANGES_REQUESTED** [review]
Hi, thanks for your example ! I made some suggestions inline to hopefully make the text a bit clearer, please check my suggestions and push the changes.

**Comment by ssipflow** [comment]
Thank you for your suggestion.!!

**Pull request closed without merging**